quasar/main.tf

```
@@ -615,3 +615,60 @@ module "storage" {
   acl        = "private"
   versioning = true
 }
+
+# Fivetran logging settings based on https://fivetran.com/docs/logs/cloudwatch/setup-guide
+# and https://github.com/tropos-io/tf-fivetran-cloudwatch
+
+data "aws_iam_policy_document" "fivetran_cloudwatch_integration_assume_role" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "AWS"
+      identifiers = ["arn:aws:iam::834469178297:root"]
+    }
+
+    condition {
+      test     = "StringEquals"
+      variable = "sts:ExternalId"
+
+      values = [
+        "${var.fivetran_cloudwatch_integration_external_id}",
+      ]
```

Since we're not doing any string interpolation here, the wrapping double-quotes can be removed. While these will technically work, they'll be flagged as "warnings" on our Terraform builds, like this:

```
Warning: Interpolation-only expressions are deprecated

  on main.tf line 659, in resource "aws_iam_policy" "fivetran_cloudwatch_integration":
 659:   policy = "${data.aws_iam_policy_document.fivetran_cloudwatch_integration.json}"

Terraform 0.11 and earlier required all non-constant expressions to be
provided via interpolation syntax, but this pattern is now deprecated. To
silence this warning, remove the "${ sequence from the start and the }"
sequence from the end of this expression, leaving just the inner expression.

Template interpolation syntax is still used to construct strings from
expressions when the template includes multiple interpolation sequences or a
mixture of literal strings and interpolations. This deprecation applies only
to templates that consist entirely of a single interpolation sequence.

(and 3 more similar warnings elsewhere)
```

```suggestion
      values = [
        var.fivetran_cloudwatch_integration_external_id,
      ]
```